setup.py

```diff
@@ -40,7 +40,7 @@
         "pypdf>=3.13.0",
         "itsdangerous>=2.1.2",
         "govuk-bank-holidays>=0.14",
-        "boto3>=1.34.100",
+        "boto3[crt]>=1.34.100",
```

Comment:
i also wonder about whether this `crt` should be here or should just be an override on doc download. unsure

Reply:
According to this there are some performance benefits to using `crt` so maybe we should have it everywhere:
https://aws.amazon.com/blogs/storage/improving-amazon-s3-throughput-for-the-aws-cli-and-boto3-with-the-aws-common-runtime/